### PR TITLE
Use explicit imports instead of library.add

### PR DESCRIPTION
**Pull request by knegusen**
There is an issue that icons are not visible in the github page. Using explicit imports instead of library could possibly fix the problem.

### CODE REVIEW

File: src/components/ui/buttons/Button.tsx [hunk 1]
```diff
@@ -18,8 +16,6 @@ import { Space } from '../layout/Space';
 import { ProgressIndicator } from '../progress/ProgressIndicator';
 import { ButtonTheme } from './ButtonTheme';
 
-addIcons(faCheck);
```
Comment (mattias800) [inline src/components/ui/buttons/Button.tsx]: This needs to be added to the imports.
Replace `<Icon name={'check'}/>` with `<Icon name={faCheck}/>`?

File: stories/initFontAwesome.tsx [hunk 2]
```diff
@@ -1,8 +1,4 @@
 import { library } from '@fortawesome/fontawesome-svg-core';
-import { fab } from '@fortawesome/free-brands-svg-icons';
-import { faCheck } from '@fortawesome/free-solid-svg-icons/faCheck';
-import { faCoffee } from '@fortawesome/free-solid-svg-icons/faCoffee';
-import { faFire } from '@fortawesome/free-solid-svg-icons/faFire';
 import { faPaw } from '@fortawesome/free-solid-svg-icons/faPaw';
 
-library.add(fab, faCoffee, faCheck, faFire, faPaw);
+library.add(faPaw);
```
Comment (mattias800) [inline stories/initFontAwesome.tsx]: This can be replaced as well?

File: src/components/ui/buttons/Button.tsx (hunk 1 above)
Comment (lindskogen) [reply src/components/ui/buttons/Button.tsx]: Yes!

File: stories/initFontAwesome.tsx (hunk 2 above)
Comment (knegusen) [reply stories/initFontAwesome.tsx]: Slarvigt av mig. Denna ska ju bort!

**Pull request merged**